src/Modals/__tests__/Modal.test.tsx

```diff
@@ -50,6 +61,60 @@ describe('Modal', function () {
         strictEqual(modal.getAttribute('aria-labelledBy'), 'facultyEditButton');
       });
     });
+    context('when forwardRef prop is present', function () {
+      beforeEach(function () {
+        const RefExample = () => {
+          const ref = useRef<HTMLDivElement>(null);
+          const [modalVisible, setModalVisible] = useState(false);
+          const onButtonClick = () => {
+            setModalVisible(true);
+            if (ref.current) {
+              ref.current.focus();
```

Comment:
The ref is always null therefore ref.current is always false and  ref.current.focus(); will never be executed.